crates/bevy_sprite/src/texture_atlas.rs

```diff
@@ -149,6 +129,26 @@ impl TextureAtlasLayout {
     }
 }
 
+/// Component used to draw a specific section of a texture.
```

Comment:
This seems to just be moving `TextureAtlas` below `TextureAtlasLayout` in the file. Merge artifact?

Reply:
Also thinking this is a drive-by cleanup, so nevermind.

Reply:
Removing this change may make the PR easier to maintain, but feel free to resolve this either way.